### DISCUSSION TITLE
Regarding Accuracy Challenge mod

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#22546] [Ideas]
With the option in the skin layout editor to show the max accuracy that is still obtainable, i think it would be nice to have an option for the accuracy challenge mod to use the max accuracy one can get to check instead of the current accuracy.
Also with the current system you always fail if the very first note is a 100, and since on some beatmaps it is very hard to judge the correct timing of the first note this makes the mod sort of useless on those beatmaps.

**Comment by monochrome22** [comment]
Related #18402 
Discussion starts here: https://github.com/ppy/osu/pull/18402#issuecomment-1386707533

Continuing the discussion from the pull thread:
I disagree with adding it as a toggle, it should be the default. Having an unpractical function as the default seems really strange. "Fail if falls below" should be the toggle option. 
I'll elaborate on why it should be the default if necessary. 

If confusion is a problem then, you could have a toggle in the skin editor for maximum achievable accuracy for "Display with Accuracy Challenge mod", which would be enabled by default. 
If this is enabled, it should also display text "Maximum achievable accuracy". 
![max](https://user-images.githubusercontent.com/122969285/234564595-0f5d5d06-4234-49f7-b8cf-4a9a1783c181.png)
Or "Final accuracy" 
![final](https://user-images.githubusercontent.com/122969285/234564679-97f0e2db-c6b6-4174-bc77-33009d5041b8.png)




"Minimum Accuracy" could also be rephrased to "Minimum final accuracy" and mod description to "Finish with a certain accuracy or fail!" At this point it would be clear even to new players how this mod works. 

As I wrote in Discord, another option is to add an explanation to the fail screen. Instead of "you failed, try again?", it could say "You failed because you can no longer finish with 98% accuracy."

Also consider my suggestions of allowing more precise values and setting 99.99% max. They are certainly useful and it's only a matter of time until they also become requested.

**Reply by SevenChords** [reply]
the maximum achievable accuracy display is already a toggle in the skin layout editor
i think we should at the very least have the option to have the accuracy challenge mod only fail if that maximum achievable falls below the set accuracy, because right now it always fails if your current accuracy falls below the set accuracy, and to allow a 100 on the first note in a song you would have to set the accuracy in the mod customisation to ~30%, and in that case the mod becomes useless, which is almost always the case on songs that have the first note be the first note of the music, with nothing but visuals to estimate the timing.

**Reply by monochrome22** [reply]
I think you misunderstood what I meant. I was replying to peppy's proposal of adding this as a toggle and his concern of it being confusing. I hope the edited comment is more clear.
I support your idea, I made this same suggestion in the pull thread https://github.com/ppy/osu/pull/18402#issuecomment-1386707533

**Reply by cdwcgt** [reply]
ehh https://github.com/ppy/osu/pull/23317
